src/d3d12/D3D12.cpp

```diff
@@ -41,6 +38,8 @@ LRESULT D3D12::OnWndProc(HWND ahWnd, UINT auMsg, WPARAM awParam, LPARAM alParam)
 
         if (d3d12.m_trapInputInImGui) // TODO: look into io.WantCaptureMouse and io.WantCaptureKeyboard
         {
+            ImGui_ImplWin32_WndProcHandler(ahWnd, auMsg, awParam, alParam);
```

Comment:
Why move this down here and ignore the result?

Reply:
One thing that isnt good though is the return, that has to happen to block input passthrough to the game. Some change that we did in imgui_impl_win32 likely causes some issues due to which you had to remove the return.

But yeah have to overall check as mentioned